src/openms/include/OpenMS/CONCEPT/Constants.h

```diff
@@ -258,13 +258,148 @@ namespace OpenMS
     */
     extern OPENMS_DLLAPI const std::string   PRECURSOR_ERROR_PPM_USERPARAM;
 
+
+    /** User parameter name for monoisotopic peak misassignment correction of a precursor (usually an integer x with the correction being x times C13C12_MASSDIFF_U)
+            String
+    */
+    extern OPENMS_DLLAPI const std::string   PRECURSOR_ASSIGNMENT_CORRECTION;
```

Comment:
also add _USERPARAM suffix for all of them?
Another idea would be to put all USER_PARAM constants into their own namespace
Constants::UserParams::FRAGMENT_ANNOTATION

Reply:
Should I make an additional class for these altogether? They do not really fit with the other constants.

Reply:
something like Names.h or UserParamNames.h

Reply:
hmm I think it is fine to just put the extra namespace

namespace OpenMS
{
  namespace Constants
  {
   namespace UserParams
    {

in the same file.
@cbielow or @jpfeuffer what is your opinion on that?